lib/urls.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
   project: {
     list: '/e/:establishmentId/projects',
     read: '/e/:establishmentId/projects/:projectId',
+    import: '/e/:establishmentId/projects/import',
```

Comment:
Is this page automatically mounted then?

Reply:
it's mounted in pages

Reply:
https://github.com/UKHomeOffice/asl-pages/blob/master/pages/project/index.js#L51